README.md

```diff
@@ -32,12 +32,21 @@ To allow for more complex configurations, you can also add a top-level section `
         },
         "ENV" : {
             "test" : {
-                "version": "1.5.0"
+                "version": "1.5.0",
+                "override": false
             }
             "win32" : {
                 "electron" : {
                     "filePattern": "electron.exe$"
+                },
+                "electron": {
```

Comment:
this should just be rolled into the above `electron` key no? otherwise we have it dupedl